Ginger/GingerCoreCommon/Repository/NewRepositorySerializer.cs

```diff
@@ -499,6 +501,7 @@ public RepositoryItemBase DeserializeFromFile(string FileName)
             if (FileName.Length > 0 && File.Exists(FileName))
             {
                 string xml = File.ReadAllText(FileName);
+                //xml = Regex.Replace(xml, "&#x([0-8BCEFbcef]|1[0-9A-Fa-f]);|[\x01-\x08\x0B\x0C\x0E\x0F\u0000-\u0008\u000B\u000C\u000E-\u001F]", "");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5038055)